RallyHereIntegration/Source/RallyHereIntegration/Private/RH_LocalPlayerSubsystem.cpp

```diff
@@ -245,6 +246,22 @@ void URH_LocalPlayerSubsystem::OnUserChanged()
 	}
 }
 
+void URH_LocalPlayerSubsystem::OnUserRefreshTokenExpired(FSimpleDelegate CompletionDelegate)
+{
+	// attempt an autologin
+	if (LoginSubsystem != nullptr)
+	{
+		LoginSubsystem->SubmitAutoLogin(false, false, false, FRH_OnLoginComplete::CreateWeakLambda(this, [this, CompletionDelegate](const FRH_LoginResult&) {
```

Comment:
This won't be safe to call here, since some OSSes are inconsistent between autologin and normal login.  We should probably create a "RetryLogin" function on the login subsystem that resubmits the last login type that was used (either autologin or login with whatever credentials).